cofi-share/src/main/java/com/omelan/cofi/share/model/Recipe.kt

```diff
@@ -26,6 +26,7 @@ enum class RecipeIcon(@DrawableRes val icon: Int, @StringRes val nameResId: Int)
     Siphon(R.drawable.recipe_icon_siphon, R.string.name_siphon),
     Bripe(R.drawable.recipe_icon_bripe, R.string.name_bripe),
     Cezve(R.drawable.recipe_icon_cezve, R.string.name_cezve),
+    CleverDripper(R.drawable.recipe_icon_drip, R.string.prepopulate_clever_dripper_name),
```

Comment:
We shouldn't add new icons if they use the same drawable. In this case `drip` icon will show up twice in the icon picker

Maybe in the future I could hire [Hubert](https://tereszkiewicz.tumblr.com/) again for new icons then I'll add Clever Dripper here.

Reply:
Maybe it would be best to change `V60` icon into a generic `drip` icon, as there are a lot of similar looking drips on the market and it makes no sense to create an icon for every single one of them. WDYT?